deu/paris-match-cleared.vbs

```diff
@@ -90,26 +112,26 @@ DO
 	'PC Person Closed, Not PARIS Interstate
 	'CC Case Closed, Not PARIS Interstate
 	EMReadScreen INTM_period, 5, row, 59
-	IF INTM_match_status = "" THEN script_end_procedure_with_error_report("A pending PARIS match could not be found. The script will now end.")
-	'IF INTM_match_status <> "RV" THEN
-	    INTM_info_confirmation = MsgBox("Press YES to confirm this is the match you wish to act on." & vbNewLine & "For the next match, press NO." & vbNewLine & vbNewLine & _
+	IF INTM_match_status = "  " THEN script_end_procedure_with_error_report("A pending PARIS match could not be found. The script will now end.")
+	IF INTM_match_status = "RV" THEN
+        row = row + 1 'skipping verified cases
+    Else
+        INTM_info_confirmation = MsgBox("Press YES to confirm this is the match you wish to act on." & vbNewLine & "For the next match, press NO." & vbNewLine & vbNewLine & _
         "   " & INTM_period, vbYesNoCancel, "Please confirm this match")
-		IF INTM_info_confirmation = vbNo THEN
+	    IF INTM_info_confirmation = vbNo THEN
             row = row + 1
-            'msgbox "row: " & row
             IF row = 18 THEN
```

Comment:
This might be needed in the row = row + 1 on line 117, it could hit the end of the list and need to page down. It should be safe to move outside of the ELSE because if vbYes is hit - the row should not increment and so won't hit 18.

Reply:
The update you made should do what is needed